### PR TITLE
Updated the coc

### PR DESCRIPTION
**Pull request by Sloth-Panda**
correponds to issue -> #5

### CODE REVIEW

File: Code_of_Conduct.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at their [discord](https://discord.com/channels/731001496417730611/790974003011518474). All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
```
Comment (praveenscience) [inline Code_of_Conduct.md:48]: Single Lines Please...

File: Code_of_Conduct.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Contributor Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment includes:
+
```
Comment (praveenscience) [inline Code_of_Conduct.md:10]: Remove 1 Empty Line.

File: Code_of_Conduct.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Examples of unacceptable behavior by participants includes:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a  professional setting
+
+
```
Comment (praveenscience) [inline Code_of_Conduct.md:26]: Remove 1 Empty Line.

File: Code_of_Conduct.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Contributor Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (praveenscience) [inline Code_of_Conduct.md:5]: This one's still not rephrased. It's alright, but which parts have you rephrased, can you please tell?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix these and let's merge this...

File: Code_of_Conduct.md (hunk 1 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:48]: you want me to remove the paragraph? or keep it as it and make everything single line

File: Code_of_Conduct.md (hunk 3 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:26]: ok

File: Code_of_Conduct.md (hunk 4 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:5]: I have rephrased all the parts except this part as it contains specific words as you can see so I just changes 2/3 words in this section only

**Review by praveenscience — APPROVED** [review]
Looks good to me! 👍

File: Code_of_Conduct.md (hunk 1 above)
Comment (praveenscience) [reply Code_of_Conduct.md]: You did it right.

**Comment by praveenscience** [comment]
Mentors, please check. When merging make sure you specify the Fixes #5 on this.

**Comment by Sloth-Panda** [comment]
@praveenscience  sir can you also add swoc to it

**Comment by Sloth-Panda** [comment]
@praveenscience  can you please check this one I see, you assigned me again on the same issue but it's pr is here

**Comment by praveenscience** [comment]
@Sloth-Panda Yea, for some reason, I told you're assigned but really didn't "assign" you the issue. Now I assigned it and as you can see, it's great. Just waiting for one more review from either of the mentors. Reach out to them and get your approval and points bro! 😁

**Comment by Sloth-Panda** [comment]
@Salonii02  and @manudd1011 can you guys please review this

**Review by Salonii02 — APPROVED** [review]

**Comment by Sloth-Panda** [comment]
@praveenscience  does it require 3 members for approval?

**Comment by praveenscience** [comment]
@Sloth-Panda 2 of them. I am extremely busy, so don't worry, I am merging it now. Have patience, we're on different time zones and I am in a full time job. 😊

**Comment by praveenscience** [comment]
@Sloth-Panda Did you see the error message what it says? 🤔

**Comment by Sloth-Panda** [comment]
No , I don't see any error messages. Where do you see though?

**Comment by praveenscience** [comment]
> does it require 3 members for approval?

Why did you ask this @Sloth-Panda?

**Comment by Sloth-Panda** [comment]
As I was trying to reachout to maintainers so wanted to know how many maintainers should I reach out to for reviewing this pr as it was pending for nearly 10 days @praveenscience

**Comment by praveenscience** [comment]
@Sloth-Panda Just one man. It says there, 2 approvals are required. One from me and one from one of the mentors.

**Pull request merged**